internal/service/controller_service.go

```diff
@@ -332,66 +284,70 @@ func (s *ControllerService) Status(req *pb.StatusRequest, stream pb.ControllerSe
 
 	defer watcher.Stop()
 
-	for result := range watcher.ResultChan() {
-		switch result.Type {
-		case watch.Added, watch.Modified, watch.Deleted:
-			exporter = result.Object.(*jumpstarterdevv1alpha1.Exporter)
-			leased := exporter.Status.LeaseRef != nil
-			leaseName := (*string)(nil)
-			clientName := (*string)(nil)
-
-			original := client.MergeFrom(exporter.DeepCopy())
-			if false ||
-				meta.SetStatusCondition(&exporter.Status.Conditions, metav1.Condition{
-					Type:               string(jumpstarterdevv1alpha1.ExporterConditionTypeOnline),
-					Status:             metav1.ConditionTrue,
-					ObservedGeneration: exporter.Generation,
-					Reason:             "Connect",
-					Message:            "Exporter is connected to the Status endpoint",
-				}) ||
-				meta.SetStatusCondition(&exporter.Status.Conditions, metav1.Condition{
-					Type:               string(jumpstarterdevv1alpha1.ExporterConditionTypeRegistered),
-					Status:             metav1.ConditionTrue,
-					ObservedGeneration: exporter.Generation,
-					Reason:             "Connect",
-					Message:            "Exporter is connected to the Status endpoint",
-				}) {
-				if err = s.Client.Status().Patch(ctx, exporter, original); err != nil {
-					logger.Error(err, "unable to update exporter status")
+	ticker := time.NewTicker(time.Second * 10)
+
+	defer ticker.Stop()
+
+	online := func() {
+		original := client.MergeFrom(exporter.DeepCopy())
```

Comment:
The only thing with this is that we will increase the pressure in reconciliation. But that's ok for now I think. We will have to keep an eye on the reconciliation loop CPU usage.

Reply:
Well, and increase the chances of conflict when updating the exporter.

Are we trying to fix a bug with the change?